src/super_gradients/training/datasets/detection_datasets/coco_format_detection.py

```diff
@@ -68,7 +72,7 @@ def _setup_data_source(self) -> int:
         """
 
         self.coco = self._init_coco()
-        self.class_ids = sorted(self.coco.getCatIds())
+        self.class_ids = sorted(cls_id for cls_id in self.coco.getCatIds() if cls_id not in self.class_ids_to_ignore)
```

Comment:
Do we want this feature?
It is useful for roboflow because we want to drop index 0, but at the same time it is not common at all so I feel like we are kind of overfitting the base class to the need of the roboflow class.
But I also don't have a better solution, so maybe we should just keep this feature like that.

Reply:
Probably we can keep it here for now. Dropping some classes or merging classes (Are recently requested) could be helpful feature to have. So we may get back to this in future.